test/multiverse/suites/high_security/high_security_test.rb

```diff
@@ -222,7 +222,7 @@ def test_doesnt_block_intrinsic_attributes_on_errors
   end
 
   def test_blocks_log_capture
-    Logger.new(StringIO.new()).fatal("Ooops")
+    Logger.new(StringIO.new()).fatal("Oops")
```

Comment:
Happy to drop, but the extra `o` didn't appear to add value

Reply:
The change works! No need to drop.